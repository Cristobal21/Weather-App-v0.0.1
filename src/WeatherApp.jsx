import { useState } from "react"
import { AddNewPlace } from "./components/AddNewPlace"
import { PlaceGrid } from "./components/PlaceGrid"
// import { WeatherGrid } from "./components/weather/WeatherGrid"



function WeatherApp() {

  const [lugares, setLugares] = useState(['miami'])

  const onAddPlace = (nuevoLugar) => {
    setLugares([nuevoLugar])
    // setLugares([ nuevoLugar, ...lugares ])
  }

  return (
    <>
      <h1 className="title-app mb-5 mt-5" >Wheater App v0.0.1</h1>

      <AddNewPlace
        onNewPlace={onAddPlace}
      />

      {
        lugares.map((lugar) => (
          
          <PlaceGrid
            key={lugar}
            lugar={lugar}
          />
            
        ))
      }      
    </>
  )
}

export default WeatherApp
