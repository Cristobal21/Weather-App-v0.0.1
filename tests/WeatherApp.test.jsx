import { render, screen } from "@testing-library/react"
import WeatherApp from "../src/WeatherApp"


describe('Pruebas en <WeatherApp/>', () => {

    test('Debe hacer match con el snapshot', () => {

        const { container } = render(<WeatherApp/>)
        expect( container ).toMatchSnapshot()
        // screen.debug()
    })


})