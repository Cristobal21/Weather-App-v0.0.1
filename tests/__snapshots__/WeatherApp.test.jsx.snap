// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <WeatherApp/> Debe hacer match con el snapshot 1`] = `
<div>
  <h1
    class="title-app mb-5 mt-5"
  >
    Wheater App v0.0.1
  </h1>
  <form
    aria-label="form"
    class=""
  >
    <div>
      <input
        class="form-control"
        placeholder="Busca un lugar, país o ciudad"
        type="text"
        value=""
      />
    </div>
  </form>
  <p
    class="mt-3"
  >
    Última búsqueda realizada: miami
  </p>
  <hr />
  <div />
</div>
`;
